tests/test_main.py

```diff
@@ -0,0 +1,25 @@
+from typer.testing import CliRunner
+
+runner = CliRunner()
+
+
+def test_gatorgrade_runs_correct():
```

Comment:
Instead of using a word like `correct`/`correctly`, it can be more semantically readable if you use the actual activity it should complete; i.e. this test should be named `test_gatorgrade_runs`.